alfalfa_web/Dockerfile

```diff
@@ -4,7 +4,7 @@ RUN apt-get update && apt-get install -y \
     ca-certificates \
     git \
     vim \
-    && rm -rf /var/lib/apt/lists/* \
+    && rm -rf /var/lib/apt/lists/*
 
 RUN cd /srv
```

Comment:
I think we can also remove the `RUN cd /srv` command as we're following that with a `WORKDIR` command so it literally does nothing